test/v0/test_api.py

```diff
@@ -38,6 +38,7 @@ def test_list_buckets(self):
             'bucketInfo': {},
             'corsRules': [],
             'lifecycleRules': [],
+            'options': set(),
```

Comment:
This change should be done in `test/v1/test_api.py:test_list_buckets` too

Reply:
Unfortunately, the `delete_bucket` api being used by `test/v1/test_api.py:test_list_buckets` doesn't have a return value/it's `None`.  So the existing test case is valid according to the existing API in `/api.py`.

Would we want to change `/api.py:delete_bucket()` to return the dictionary from `/raw_api.py:delete_bucket()`?  
Was there a reason the old API returned the value from `delete_bucket` and the new one doesn't?  What do you think?

Reply:
Oh, you are right!

I think it was listed in a comment as something that is not needed. Should the user of `delete_bucket()` expect to receive a dict/json in return to his (successful) call? Why?

Reply:
Looks like you updated it back in June.  I think it works fine as-is.  If you're good with it we can mark this resolved and merge.